pkg/vmimage/data/script.sh

```diff
@@ -164,7 +172,29 @@ yum -y install \
     tree \
     find \
     WALinuxAgent \
-    yum-utils
+    yum-utils \
+    azure-mdsd \
+    azure-security \
+    azsec-monitor \
+    td-agent
+
+# td-agent needs access to the journal
+usermod -a -G systemd-journal td-agent
+
+# openshift-audit dir ends up with wrong permissions, preemptively fix that
+mkdir -pv /var/log/openshift-audit
+chmod -v 0755 /var/log/openshift-audit
+
+# we need to build gems
+yum -y groups install "Development Tools"
```

Comment:
Can we narrow this down a bit to just the build tools we need? This is a pretty large metapackage and I doubt that ruby needs e.g. doxygen.

Reply:
installing from the yum repos means not having to do this now, let's see if that's possible.